terraform/init.tf

```diff
@@ -11,11 +11,6 @@ terraform {
 }
 
 locals {
-  // Unclear where this value comes from; presumably it's a deprecated value from AWS. -Aidan Feldman, 11/6/17
-  old_cloudfront_zone_id = "Z33AYJ8TM3BH4J"
-
-  // https://docs.aws.amazon.com/general/latest/gr/rande.html#cf_region
-  cloudfront_zone_id = "Z2FDTNDATAQYW2"
```

Comment:
This deletion is the main change. Switched every reference to it to the local defined in `cloud.gov.tf`.